### DISCUSSION TITLE
Question: Master HA / High Availability in v3.x community

### DISCUSSION THREAD

**Discussion by jkiebzak** [moosefs/moosefs#387] [Q&A]
Was Master HA / High Availability / leader-follower ever released to the community?

At least these 2 issues mention HA will be released:
- [#85 (comment from May 20, 2018)](https://github.com/moosefs/moosefs/issues/85#issuecomment-390521097)
- [#39 comment from Mar 16, 2018](https://github.com/moosefs/moosefs/issues/39#issuecomment-373622547)

If HA **has** been released to the community, how do you start Master in HA?

If HA **has not** been released to the community, will it?

**Comment by chogata** [comment]
We are currently in process of preparing the next major (4) version of MooseFS for Community Edition release. It has new features that we are sure will be of use to our community users. We are not sure yet if HA will make it to this release or not, however, we feel that with the current system of metaloggers MooseFS is as secure as it can be.

**Comment by thenitai** [comment]
The issue is not "as secure as it can be" but availability for clients. As far as I can see, this is only available in the Pro version. No problem with that, but is there some price for small instances?

**Comment by chogata** [comment]
@thenitai Please, contact the team that handles the Pro version at contact@moosefs.pro - I'm quite sure they will be able to find a solution that will suit you.
